test/verify/check-image

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/python3
+
+# checkpoint:
+# 1. create supported image
+
+
+import parent
+import testlib
+
+
+@testlib.nondestructive
+@testlib.timeout(2400)
```

Comment:
always better to have this only for command which may actually take that long as when things get really broken (like some new changes and tests needs adjusting) then debugging test with such timeout is "fun" :)
You can do:
```
<command that should be quick>
with b.wait_timeout(2400):
   <command1 that takes long>
   <command2 that thakes long>
<command that should be also quick>
```

Reply:
From the [code](https://github.com/cockpit-project/cockpit/blob/9b786edbec00af2164f7e91df61329cc33044096/test/common/testlib.py#L1375), the `@testlib.timeout()` is different from `browser.wait_timeout()`. The first one is the whole test(method)/suite(class) scope. Since the image building needs take a long time, so I have to set the test scope timeout. And it should be bigger than `browser.wait_timeout()` which I configured in [test](https://github.com/osbuild/cockpit-composer/pull/931/files#diff-52613cf5f87fb672c28b7c6b27f4e655R126).
So I set `browser.wait_timeout()` to `30 minutes` and `@testlib.timeout()` to `40 minutes`.

Reply:
Oh, of course, you are right. Please ignore :)